pkg/wallet/util.go

```diff
@@ -161,9 +161,10 @@ func addInsAndOutsToPset(
 	return ptx.ToBase64()
 }
 
-func extractScriptTypesFromPset(ptx *pset.Pset) ([]int, []int, []int, []int) {
+func extractScriptTypesFromPset(ptx *pset.Pset) ([]int, []int, []int, []int, []int) {
```

Comment:
should we move this to esitamtion.go ?

Reply:
I put these functions in util because they're used in transasction.go